python/federatedml/secret_sharing_sum/secret_sharing_sum_host.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+#  Copyright 2019 The FATE Authors. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+
+from federatedml.util import LOGGER
+from federatedml.transfer_variable.transfer_class.secret_sharing_sum_transfer_variable import \
+    SecretSharingSumTransferVariables
+from federatedml.param.secure_sharing_sum_param import SecureSharingSumParam
+from federatedml.secret_sharing_sum.base_secret_sharing_sum import BaseSecretSharingSum
+
+
+class SecretSharingSumHost(BaseSecretSharingSum):
+    def __init__(self):
+        super(SecretSharingSumHost, self).__init__()
+        self.transfer_inst = SecretSharingSumTransferVariables()
+        self.host_party_idlist = []
+        self.local_partyid = -1
+
+    def _init_model(self, model_param: SecureSharingSumParam):
+        self.sum_cols = model_param.sum_cols
+
+    def _init_data(self, data_inst):
+        self.local_partyid = self.component_properties.local_partyid
+        self.host_party_idlist = self.component_properties.host_party_idlist
+        self.host_count = len(self.host_party_idlist)
+        self.vss.set_share_amount(self.host_count+1)
+        if not self.sum_cols:
+            self.x = data_inst.mapValues(lambda x: x.features)
```

Comment:
this part gives a numpy array, while select_data_by_idx gives a list. In general, we keep table entries as Instance objects for convenience; attribute "features" of data instance objet is always a numpy array to allow fast math operation.